### DISCUSSION TITLE
confidence intervals and total (not just non-zero cell) counts

### DISCUSSION THREAD

**Discussion by ejagoda14** [Katsevich-Lab/sceptre#116] [Q&A] [answered]
-Hi there, I have two questions:

1. Is there a way to extract a confidence interval or some other sense of the reliability of the effect size estimate from the sceptre object? 
2. In the discovery results object, there is a count given of the number of perturbed and control cells with non-zero counts for the gene. With my understanding, cells with zero counts for the gene that have guides for the target are still included in the differential expression calculation, is that correct? And if so, is there a way to report the actual number of perturbed cells used in the differential expression analysis?

**Comment by timothy-barry** [comment, marked as answer]
Hi Evvie,

1. Unfortunately no, sceptre does not currently return confidence intervals. This turns out to be a fairly challenging statistical problem. However, you could determine whether the confidence interval would cover zero by looking at the p-value. For example, if the p-value is below 0.05, then the 95% confidence interval would not cover zero.
2. > cells with zero counts for the gene that have guides for the target are still included in the differential expression calculation, is that correct?

Yes, your interpretation is correct. To determine the actual number of perturbed cells used in the DE test (in high-MOI), you could do the following:
```
sceptre_object@grna_assignments$grna_group_idxs |> sapply(length)
```
And the total number of cells used in the DE test is just the number of cells remaining after cellwise QC.

**Comment by ekatsevi** [comment]
An update on the first question: since sceptre 0.10.3, association analysis results include `se_fold_change`, the standard error of the fold-change estimate, alongside `fold_change`. The answer above regarding cell counts remains unchanged.
